### PR TITLE
document `.yaml.local`

### PR DESCRIPTION
**Pull request by mmetc**
Document for https://github.com/crowdsecurity/crowdsec/pull/1497

### CODE REVIEW

File: crowdsec-docs/docs/configuration/crowdsec_configuration.md [hunk 1]
```diff
@@ -105,6 +104,57 @@ export DB_PASSWORD="<db_password>"
...
+**Note:** you cannot remove configuration keys from a `.local` file, only
+change them (possibly with an empty or default value). So for example, removing
+`db_config.db_path` is not possible, even if you don't use it. And you cannot
+remove a whole mapping (like `api.server`). Sequences on the other hand, are
+always replaced.
+:::
+
+### Configuration files that support `.yaml.local`:
+
+- `config.yaml`
```
Comment (buixor) [inline crowdsec-docs/docs/configuration/crowdsec_configuration.md:149]: I think we might want to add :
 - `acquis.yaml`
 - `console.yaml`
 - `notification/*.yaml` (but this one can be more tricky)

File: crowdsec-docs/docs/profiles/intro.md [hunk 2]
```diff
@@ -4,11 +4,17 @@ title: Introduction
...
- - increase or decrease decisions `duration` (default: `4h`)
- - decide the type of remediation that should be applied (default: `ban`)
- - decide of the scope of the remediation (default: `ip`)
- - enable/disable debug for a specific profile
+The profiles configuration allows users to configure which kind of remediation should be applied when a scenario is triggered. The profile can be used to:
 
+- increase or decrease decisions `duration` (default: `4h`)
+- decide the type of remediation that should be applied (default: `ban`)
+- decide the scope of the remediation (default: `ip`)
+- enable/disable debugging for a specific profile
```
Comment (buixor) [inline crowdsec-docs/docs/profiles/intro.md:12]: maybe we might want to reference notifications here (even if it's not related to your changes directly)

**Review by buixor — CHANGES_REQUESTED** [review]
some suggestions, but not necessarily directly related to your changes. What do you think ?

File: crowdsec-docs/docs/configuration/crowdsec_configuration.md (hunk 1 above)
Comment (mmetc) [reply crowdsec-docs/docs/configuration/crowdsec_configuration.md:149]: it should not be a problem for config or plugins - but are you sure for acquis.yaml ? it already has acquisition_dir

**Review by blotus — APPROVED** [review]

**Pull request merged**